### PR TITLE
i18n(es): Translated `mdx.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `mdx.mdx` to spanish

### CODE REVIEW

**Comment by Waxer59** [comment]
I have fixed a file that had broken links pointing to this page.

File: src/content/docs/es/core-concepts/astro-pages.mdx [hunk 1]
```diff
@@ -61,7 +61,7 @@ import MySiteLayout from '../layouts/MySiteLayout.astro';
 
 ## Páginas Markdown/MDX
 
-Astro trata archivos Markdown (`.md`) dentro de `src/pages/` como páginas en tu proyecto. Si tienes la [integración de MDX instalada](/es/guides/integrations-guide/mdx/#installation), también procesa los archivos MDX (`.mdx`) de la misma manera. Estos se usan comúnmente para páginas con mucho texto, como artículos de blog y documentación.
+Astro trata archivos Markdown (`.md`) dentro de `src/pages/` como páginas en tu proyecto. Si tienes la [integración de MDX instalada](/es/guides/integrations-guide/mdx/#instalación), también procesa los archivos MDX (`.mdx`) de la misma manera. Estos se usan comúnmente para páginas con mucho texto, como artículos de blog y documentación.
```
Comment (dreyfus92) [inline src/content/docs/es/core-concepts/astro-pages.mdx:64]:
```suggestion
Astro trata los archivos Markdown (`.md`) dentro de `src/pages/` como páginas en tu proyecto. Si tienes la [integración de MDX instalada](/es/guides/integrations-guide/mdx/#instalación), también procesa los archivos MDX (`.mdx`) de la misma manera. Estos se usan comúnmente para páginas con mucho texto, como artículos de blog y documentación.
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Te dejo una sugerencia @Waxer59, buen trabajo 😁

**Comment by Waxer59** [comment]
Gracias por la revisión 🤠

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Nice job peeps, 💜

**Pull request merged**